staging/src/k8s.io/kubectl/pkg/cmd/certificates/certificates.go

```diff
@@ -73,14 +57,33 @@ type CertificateOptions struct {
 	genericclioptions.IOStreams
 }
 
-// NewCertificateOptions creates the options for certificate
+// NewCertificateOptions creates CertificateOptions struct for `certificate` command
 func NewCertificateOptions(ioStreams genericclioptions.IOStreams, operation string) *CertificateOptions {
 	return &CertificateOptions{
 		PrintFlags: genericclioptions.NewPrintFlags(operation).WithTypeSetter(scheme.Scheme),
 		IOStreams:  ioStreams,
 	}
 }
 
+// NewCmdCertificate returns `certificate` Cobra command
+func NewCmdCertificate(f cmdutil.Factory, ioStreams genericclioptions.IOStreams) *cobra.Command {
```

Comment:
minimize the diff by avoiding reordering in this PR

Reply:
Fixed